www/front_src/src/Navigation/Sidebar/Menu/CollapsableItems.tsx

```diff
@@ -236,26 +251,118 @@ const CollapsableItems = ({
     setCollapseScrollMaxWidth((window.innerWidth - rect.left) / 8);
   };
 
+  const fillItemsHoveredByDefault = (
+    rootItem: Page | null,
+    parentItem: Page | null,
+    childItem: Page | null,
+  ): void => {
+    rootItemHoveredByDefault.current = rootItem;
+    parentItemHoveredByDefault.current = parentItem;
+    childItemHoveredByDefault.current = childItem;
+  };
+
+  const searchItemsHoveredByDefault = (
```

Comment:
This function seems too complicated to me. I would try to simplify it by reduce the if conditions nesting, and also probably use extract some logic to dedicated functions.